sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOLROIT.java

```diff
@@ -38,6 +38,7 @@ public class FhirIOLROIT {
 
   private transient HealthcareApiClient client;
   private final String project;
+  private static final String BQ_DATASET_ID = "big_query_storage";
```

Comment:
I saw that BigQueryIO uses this dataset id for some of their tests, so figured we could use it here. However, I get permission denied when trying to export there. Should we request access to this dataset or should we create our own? @msbukal @pabloem